doc/api/tls.markdown

```diff
@@ -733,20 +739,20 @@ dictionary with keys:
   instead of the client preferences. For further details see `tls` module
   documentation.
 
-If no 'ca' details are given, then Node.js will use the default
+If no 'CA' details are given, then Node.js will use the default
 publicly trusted list of CAs as given in
 <http://mxr.mozilla.org/mozilla/source/security/nss/lib/ckfw/builtins/certdata.txt>.
 
 ## tls.createSecurePair([context][, isServer][, requestCert][, rejectUnauthorized][, options])
 
 Creates a new secure pair object with two streams, one of which reads/writes
-encrypted data, and one reads/writes cleartext data.
-Generally the encrypted one is piped to/from an incoming encrypted data stream,
-and the cleartext one is used as a replacement for the initial encrypted stream.
+encrypted data, and one of which reads/writes cleartext data. Generally the 
+encrypted one is piped to/from an incoming encrypted data stream, and the 
```

Comment:
No, you're correct here: this isn't an enumeration, it's just a conjunction of two clauses.